Fix flaky test_parameter_example test

Test `tests/integration_tests/test_parameter_example.py::test_parameter_example` fails occasionally on standard PR runs.
Example and more info here: https://github.com/equinor/ert/actions/runs/8222932157/job/22485039874?pr=7385

```sh
 File "/Users/runner/work/ert/ert/tests/integration_tests/test_parameter_example.py", line 321, in test_parameter_example
  |     @given(
File "/Users/runner/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/hypothesis/core.py", line 1635, in wrapped_test
  |     raise the_error_hypothesis_found
  | exceptiongroup.ExceptionGroup: Hypothesis found 2 distinct failures. (2 sub-exceptions)
```

There is still a problem with this test: https://github.com/equinor/ert/actions/runs/8359835767/job/22884212033#step:8:5940
```
FAILED tests/integration_tests/test_parameter_example.py::test_parameter_example - ExceptionGroup: Hypothesis found 2 distinct failures. (2 sub-exceptions)
```

--- a/tests/integration_tests/test_parameter_example.py
+++ b/tests/integration_tests/test_parameter_example.py
@@ -7,7 +7,6 @@ from typing import Literal, Optional
 import cwrap
 import hypothesis.strategies as st
 import numpy as np
-import pytest
 import xtgeo
 from hypothesis import given, note, settings
 from hypothesis.extra.numpy import arrays
@@ -116,7 +115,7 @@ class IoProvider:
     def _random_values(self, shape, name):
         return self.data.draw(
             arrays(
-                elements=st.floats(min_value=2.0, max_value=10.0, width=32),
+                elements=st.floats(min_value=2.0, max_value=4.0, width=32),
                 dtype=np.float32,
                 shape=shape,
             ),
@@ -327,7 +326,6 @@ class SurfaceParameter:
             )
 
 
-@pytest.mark.skip
 @settings(max_examples=10)
 @given(
     io_source=st.builds(IoProvider, data=st.data()),

